crates/ruff_python_parser/src/lib.rs

```diff
@@ -661,6 +661,11 @@ impl FusedIterator for TokenIterWithContext<'_> {}
 ///
 /// The mode argument specifies in what way code must be parsed.
 #[derive(Clone, Copy, Debug, Hash, PartialEq, Eq)]
+#[cfg_attr(
+    feature = "serde",
+    derive(serde::Deserialize),
+    serde(rename_all = "lowercase")
```

Comment:
We normally use kebab case everywhere

```suggestion
    serde(rename_all = "kebab_case")
```

Reply:
I think it's `kebab-case` (hyphen, not an underscore) (https://serde.rs/container-attrs.html#rename_all)